### PR TITLE
Show tenths of a cent on transation and invoice confirmation views.

### LINKED ISSUE #1023: Show tenths of cents on invoice confirmation page
### Description

When entering an invoice for Yalls.org, the cost of an article is $0.006, but the App rounds it to $0.01. Would be useful to show tenths of a cent for very small invoices (e.g. < $0.05)

### Steps to reproduce the behavior

Create an invoice to view an article at Yalls.org (current cost is 150 sats, or $0.006)

### Expected behavior

Show the invoice value of $0.006.

### Actual behavior

Shows the invoice value of $0.01

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #1023.

This got me thinking we should show tenths of a cent everywhere... but maybe that's too off-putting for initial launch.

### CODE REVIEW

File: src/helper.js [hunk 1]
```diff
@@ -113,15 +114,19 @@ export const calculateExchangeRate = (satoshis, settings) => {
...
   ) {
     throw new Error('Invalid input!');
   }
   return settings.displayFiat
-    ? formatFiat(calculateExchangeRate(satoshis, settings), settings.fiat)
+    ? formatFiat(
+        calculateExchangeRate(satoshis, settings),
+        settings.fiat,
+        centTenths
+      )
```
Comment (tanx) [inline src/helper.js]: Couldn't we just change `formatFiat` to prepend a `< ` if the string matches `/0[,.]00/`?

Comment (valentinewallace) [reply src/helper.js]: Do you mean if `formatFiat` is going to return `0.00`, it will instead return `<0.00`? I think I'm misunderstanding...

Comment (tanx) [reply src/helper.js]: Exactly. Why do we need to add an arg to `toAmountLabel`?

**Comment by tanx** [comment]
Closing in favor of #1049

**Pull request closed without merging**